proto/meta.proto

```diff
@@ -280,6 +289,7 @@ message SubscribeResponse {
     backup_service.MetaBackupManifestId meta_backup_manifest_id = 17;
     SystemParams system_params = 19;
     hummock.WriteLimits hummock_write_limits = 20;
+    RelationGroup relation_group = 21;
```

Comment:
I guess we can ignore the compatibility here if we're not going to support progressive upgrades in a cluster. 😁